tests/mpi/periodicity_03.cc

```diff
@@ -507,7 +500,7 @@ namespace Step22
     }
     catch (GridTools::ExcPointNotFound<dim> &p)
     {
-      pcout<< "Point: " << point << " is not inside a cell!" <<std::endl;
+      std::cout<< "Point: " << point << " is not inside a cell!" <<std::endl;
```

Comment:
I'm not particularly fond of tests writing to cout. The reason is that it clutters the output when you run `ctest -V` to see why a test is failing. Rather, informative output should go to `deallog` and every other output should simply be removed.

I know this is pre-existing for this test, so feel free to ignore this comment if you don't want to change this.